src/Raven.Server/Documents/Revisions/RevertRevisionsRequest.cs

```diff
@@ -1,10 +1,14 @@
 using System;
+using System.Collections.Generic;
+using Amqp.Types;
 
 namespace Raven.Server.Documents.Revisions
 {
     public class RevertRevisionsRequest
     {
         public DateTime Time { get; set; }
         public long WindowInSec { get; set; }
+        public bool ApplyToSpecifiedCollectionsOnly { get; set; } = false;
```

Comment:
why this is needed?

Reply:
If client will intend to revert 1 collection, then one collection will be in the array, right? I do not understand why this is needed. If list is null or empty then all collections should be reverted (previous behavior)